lib/index.js

```diff
@@ -124,13 +124,28 @@ bedrock.events.on(
  * @param identity the Identity containing at least the minimum required data.
  * @param [meta] optional meta information to include.
  *
- * @return a Promise that resolves to the new record.
+ * @return {Promise} the new record.
  */
 api.insert = brCallbackify(async ({actor, identity, meta}) => {
   assert.object(identity, 'identity');
   assert.string(identity.id, 'identity.id');
   assert.optionalString(identity.owner, 'identity.owner');
 
+  // select only allowed identity properties. It is particularly important that
+  // `sysResoureRole` is excluded in case someone upgrades to v5.x+ without
+  // updating their code.
+  identity = _.pick(identity, [
+    'id',
+    'description',
+    'email',
+    'label',
+    'memberOf',
+    'owner',
+    'sysSlug',
+    'type',
+    'url',
+  ]);
```

Comment:
Is this the behavior we want?  The above would be confusing if you try to insert an identity with other properties and they just vanish.  If we are enforcing a schema, seems a validator might be more appropriate.  However, that is certainly more heavyweight and without something like a "skipValidation" flag it could duplicate work that the web layer is doing.